pkg/cmd/root/help_topic.go

```diff
@@ -91,44 +91,96 @@ var HelpTopics = map[string]map[string]string{
+
+		The --jq flag requires a string argument in %[1]sjq%[1]s query syntax, and will only print those JSON values which match the query. %[1]sjq%[1]s queries can be used to select elements from an array, fields from an object, create a new array, and more. The %[1]sjq%[1]s utility does not need to be installed on the system to use this formatting directive. To learn about %[1]sjq%[1]s query syntax, see: <https://stedolan.github.io/jq/manual/v1.6/>
+
+		The --template flag requires a string argument in Go template syntax, and will only print those JSON values which match the query. In addition to the Go template functions in the standard library, the following functions can be used with this formatting directive:
+		- %[1]sautocolor%[1]s: like %[1]scolor%[1]s, but only emits color to terminals
+		- %[1]scolor <style> <input>%[1]s: colorize input using <https://github.com/mgutz/ansi>
+		- %[1]sjoin <sep> <list>%[1]s: joins values in the list using a separator
+		- %[1]spluck <field> <list>%[1]s: collects values of a field from all items in the input
+		- %[1]stablerow <fields>...%[1]s: aligns fields in output vertically as a table
+		- %[1]stablerender%[1]s: renders fields added by tablerow in place
+		- %[1]stimeago <time>%[1]s: renders a timestamp as relative to now
+		- %[1]stimefmt <format> <time>%[1]s: formats a timestamp using Go's Time.Format function
+		- %[1]struncate <length> <input>%[1]s: ensures input fits within length
+
+		To learn more about Go templates, see: <https://golang.org/pkg/text/template/>.
 		`, "`"),
 		"example": heredoc.Doc(`
-			# format issues as table
-			$ gh issue list --json number,title --template \
-			  '{{range .}}{{tablerow (printf "#%v" .number | autocolor "green") .title}}{{end}}'
-
-			# format a pull request using multiple tables with headers
+			# default output format
+			$ gh pr list
+			Showing 23 of 23 open pull requests in cli/cli
+
+			#123  A helpful contribution          contribution-branch              about 1 day ago
+			#124  Improve the docs                docs-branch                      about 2 days ago
+			#125  An exciting new feature         feature-branch                   about 2 days ago
+
+
+			# adding the --json flag with a list of field names
+			$ gh pr list --json number,title,author
+			[
+			  {
+				"author": {
+				  "login": "monalisa"
+				},
+				"number": 123,
+				"title": "A helpful contribution"
+			  },
+			  {
+				"author": {
+				  "login": "codercat"
+				},
+				"number": 124,
+				"title": "Improve the docs"
+			  },
+			  {
+				"author": {
+				  "login": "cli-maintainer"
+				},
+				"number": 125,
+				"title": "An exciting new feature"
+			  }
+			]
+
+
+			# adding the --jq flag and filtering the output to select the second result
```

Comment:
not sure if it makes sense to add a more complex `jq` example as well, or if that would be more confusing than it's worth

Reply:
Let's change an example to something that prints raw values of leaf nodes, rather than printing a nested JSON structure. For example: `.[].author.login` prints
```
monalisa
codercat
cli-maintainer
```

The result of such jq expression can then be piped into a line-based script such as shell loop, which makes it more useful in my mind then outputting JSON.